src/pages/ResultDetail.tsx

```diff
@@ -209,7 +213,7 @@ function ResultDetail() {
         )}
       </Grid>
       <Grid row>
-        <Grid col={12}>
+        <Grid col={12} className="print-hide">
```

Comment:
We may have to move this `print-hide` up a few levels. @jeffmaher and I were trying to debug the Safari browser issue @admoorgit found, regarding there being too much space between the `Detail` and `Services` sections in the print view. The map and 'get directions' button below it is possibly still taking up space on the print detail view.

Reply:
Went to test this approach and because of how the columns are set up, `print-hide` already exists at the uppermost level of the column. I spoke with @admoorgit this morning and he said this is not a blocker as it is only happening in Safari and the functionality still works, so going to push this up soon!